.github/workflows/publish-android.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish Breez SDK to Breez Maven Repository
+on:
+  workflow_dispatch:
```

Comment:
Currently setup to run manually[ just like the setup for the swift Package](https://github.com/breez/breez-sdk-swift). We can also set this up to run automatically whenever we do a release in the repository.